### ISSUE TITLE
./test_data_studies/commands.sh does not fully populate db

### ISSUE THREAD

**Issue by squirrelo** [biocore/qiita#1053]
In #1039 we realized that the script is not fully populating the database. We need to have at least the `preprocessed_params` and `processed_params` tables populated, and hunt for any others that are not currently being populated.

**Comment by ElDeveloper** [comment]
I'm trying to review #1039 but this problem keeps me from doing it i.e. the exception @josenavas reported, is showing for me as well. Is there a way to workaround this to be able to test the current functionality or should we just go with it as is? (broken, which is not terrible as it is not in master, but still).

**Comment by squirrelo** [comment]
For now, you can manually add the reference to `reference` table and then add a row to `processed_params_sortmerna` and `processed_params_uclust`

**Comment by ElDeveloper** [comment]
Any idea, why this wasn't happening before? It's kinda strange, I'm
creating a new environment using qiita_env make and still see the
problem.

**Comment by squirrelo** [comment]
If you make a test database and log in as test@foo.bar you can test things using htat. Only one study, but it at least populates.

**Comment by ElDeveloper** [comment]
Got it to work, I had to create a test environment and load my test data
there, more updates soon. Thanks for the help.

**Comment by antgonza** [comment]
@ElDeveloper any updates? Is this still an issue?

**Comment by josenavas** [comment]
The tables listed in the issue no longer exist - @ElDeveloper can you check if this is still an issue? If so, can you update the issue to reflect the specific problems that we may be receiving now?

**Comment by ElDeveloper** [comment]
I will after tomorrow, need my installation as is for development purposes that should meet a deadline tomorrow morning.

**Comment by ElDeveloper** [comment]
Sorry for the delay, I just checked and this seems to still be a problem, though likely not due to the same reasons, when I run:

```bash
qiita-env make --no-load-ontologies
./test_data_studies/commands.sh
```

And then I go to the study list page I get the following in the log file:
```
[E 160126 11:27:11 web:1228] Uncaught exception GET /study/search/?&user=demo@microbio.me&sEcho=446&query=&_=1453836431855 (::1)
    HTTPRequest(protocol='https', host='localhost:1111', method='GET', uri='/study/search/?&user=demo@microbio.me&sEcho=446&query=&_=1453836431855', version='HTTP/1.1', remote_ip='::1', headers={'Accept-Language': 'en-US,en;q=0.8,es;q=0.6,it;q=0.4', 'Accept-Encoding': 'gzip, deflate, sdch', 'X-Requested-With': 'XMLHttpRequest', 'Host': 'localhost:1111', 'Accept': 'application/json, text/javascript, */*; q=0.01', 'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.111 Safari/537.36', 'Connection': 'keep-alive', 'Referer': 'https://localhost:1111/study/list/', 'Cookie': 'user=ImRlbW9AbWljcm9iaW8ubWUi|1453836429|1a05306031c25531fe23c1e12f7419b0bcfef00a'})
    Traceback (most recent call last):
      File "/Users/yoshikivazquezbaeza/.virtualenvs/qiita/lib/python2.7/site-packages/tornado/web.py", line 1141, in _when_complete
        callback()
      File "/Users/yoshikivazquezbaeza/.virtualenvs/qiita/lib/python2.7/site-packages/tornado/web.py", line 1162, in _execute_method
        self._when_complete(method(*self.path_args, **self.path_kwargs),
      File "/Users/yoshikivazquezbaeza/.virtualenvs/qiita/lib/python2.7/site-packages/tornado/web.py", line 2297, in wrapper
        return method(self, *args, **kwargs)
      File "/Users/yoshikivazquezbaeza/git_sw/qiita/qiita_core/util.py", line 94, in wrapper
        return func(*args, **kwargs)
      File "/Users/yoshikivazquezbaeza/git_sw/qiita/qiita_pet/handlers/study_handlers/listing_handlers.py", line 337, in get
        proc_samples=proc_samples)
      File "/Users/yoshikivazquezbaeza/git_sw/qiita/qiita_core/util.py", line 94, in wrapper
        return func(*args, **kwargs)
      File "/Users/yoshikivazquezbaeza/git_sw/qiita/qiita_pet/handlers/study_handlers/listing_handlers.py", line 214, in _build_study_info
        proc_samples)
      File "/Users/yoshikivazquezbaeza/git_sw/qiita/qiita_core/util.py", line 94, in wrapper
        return func(*args, **kwargs)
      File "/Users/yoshikivazquezbaeza/git_sw/qiita/qiita_pet/handlers/study_handlers/listing_handlers.py", line 102, in _build_single_study_info
        for pd_id in proc_datas])
      File "/Users/yoshikivazquezbaeza/git_sw/qiita/qiita_core/util.py", line 94, in wrapper
        return func(*args, **kwargs)
      File "/Users/yoshikivazquezbaeza/git_sw/qiita/qiita_pet/handlers/study_handlers/listing_handlers.py", line 133, in _build_single_proc_data_info
        ref = Reference(params.pop('reference'))
    KeyError: 'reference'
[E 160126 11:27:12 web:1635] 500 GET /study/search/?&user=demo@microbio.me&sEcho=446&query=&_=1453836431855 (::1) 57.56ms
```

**Comment by antgonza** [comment]
Fixed in: #2514
